docs/themes/oso-tailwind/assets/css/styles.css

```diff
@@ -96,3 +96,16 @@ button > * {
   height: 1rem;
   vertical-align: middle;
 }
+
+.prose h5 + * {
```

Comment:
In theory, these should be above in the components layer, and use `@apply`.

e.g.

```
.prose h5 + * {
   @apply mt-0;
}
```

but I'm not sure how to reverse engineer the other styles from [here](https://github.com/tailwindlabs/tailwindcss-typography/blob/95bb9eb2807465432dad98d083d7573c4c5b5962/src/styles.js#L395-L398)